Lectures/Lecture1_theOpenScienceEthos.md

```diff
@@ -10,27 +10,28 @@
-In this lecture, student are introduced to various aspects of open science and how people and organizations are getting there from the current state of science. 
+Adopting and implementing open science practices in academia has entered mainstream science policy discussions for at least a decade. Despite several national and international efforts, there are still several persistent misrepresentations of the open science basic demands and main achievements. 
+In this lecture, student are introduced to a brief history of open science and how people and organizations are dealing with this global transformation of science. 
 After the introducing different schools of open science students are encouraged to take a point of view on the relation of each classification to their potential future projects and to name consequences of open science for their careers, and for science in general. 
-Besides that, the international context and access to scientific knowledge will be discussed, as well as the opposition to the inacted new research policies in connection with open science and the new system of recognition in academia. 
+Besides that, the international context and especially fair access to scientific knowledge will be discussed, as well as the opposition to the inacted new research policies in connection with the open science movement and the new system of recognition in academia. 
 
 ## Vision of outcome
 + Course participants are able to engage in an informed discussion about the requirements and achievements of the open science movement
 + Course participants are aware of the sociological barriers to adopting open science practices and the collective actions necessary to overcome those
 + Course participants can appreciate the logical reasoning behind choosing the selection of topics that are covered in this course.
 
 ## Embedding/community
-In this lecture we will discuss the advancement of the open science movement in the international and national context. We will cover the national and university-wide organizational aspects and present some disciplinary positive and negative examples from recent years in the physics community.
+In this lecture, we will discuss the advancement of the open science movement in the international and national context. We will cover the national and university-wide organizational aspects and present some disciplinary positive and negative examples from recent years in the physics community.
 
 ## Methods/exercises
 + The lecturer will present the [slides]() covering:
     - Increasing complexity of scientific challenges
     - Five schools of Open Science 
     - Current state of the open science conversation (NL, UU, Physics) 
+    - The connection between open science and research integrity
     - Access to knowledge and academic emancipation
     - Community-Discipline-Employer
     - The credibility cycle and "laboratory life"
```

Comment:
Last year, the students particularly liked the Latour discussion. Are we going to go deeper into the sociology of science this year, and if so where/how?

Reply:
yes, I will certainly discuss that in my lecture, and it is also covered in the podcast. I will try to discuss the Credibility cycle here only in the context of need for open science practices